### ISSUE TITLE
Version 3.14.0 is labeled as a virus by BitDefender

### ISSUE THREAD

**Issue by DankMemeGuy** [MalloyDelacroix/DownloaderForReddit#284]
Windows, v3.14.0. It's being labeled as a virus by BitDefdner. I don't know how to submit a false positive report, but if anyone else knows feel free to let me know or do it yourself. Thank you!

Specifically it says "Gen:Variant.Tedy.84060" is infected in the exe.

**Comment by durhambell** [comment]
Not just BitDefender. Windows Security and 12 vendors on Virus Total label it as malware. If this were from a website, I'd quarantine it.
![image](https://user-images.githubusercontent.com/9648408/156939729-de87d64e-850c-4031-bc80-19ce6dd5d8a2.png)

**Comment by e2489** [comment]
I assume it's a false positive, because the project is open source, but has anyone actually checked it out? And I'm confused as to why it hasn't been addressed. @MalloyDelacroix ?

**Comment by MalloyDelacroix** [comment]
This is a duplicate issue of #234, which has been closed.  I've treated this issue as if that one were re-opened.  Our best guess is that packaging the app with pyinstaller is leading to false positives.  I am not able to reproduce any of these errors on any of my systems, and I use BitDefender.

The only real solution that has been proposed is using a third-party service to sign the code.  This comes with some complications and drawbacks, so I have been hesitant to pursue this option.

Since I am unable to reproduce these errors, I have been unable to test and try to fix this issue.  It doesn't seem to be very prevalent.  And as you said, it's an open-source project, so I hope anyone who encounters this issue realizes that it is a false positive.  As for now, I'm frankly at a loss on how to remedy this.
